packages/unocss/src/postcss.ts

```diff
@@ -1 +1,2 @@
-export * from '@unocss/postcss'
+import postcss from '@unocss/postcss'
+export default postcss
```

Comment:
Main change is here, other changes didn't even have an effect. I just added them for consistency.